Comments/components/commentBrowser.js

```diff
@@ -25,12 +18,22 @@ export class CommentBrowser extends Window {
   }
 
   static isOpen () {
-    return instance;
+    return instance && $world.get('comment browser');
+  }
+
+  static async removeCommentForMorph (updatedComment, morph) {
+    await instance.removeCommentForMorph(updatedComment, morph);
   }
 
-  static async update () {
-    if (CommentBrowser.isOpen()) {
-      await instance.refreshCommentGroupMorphs();
+  static async addCommentForMorph (comment, morph) {
```

Comment:
gibt es einen Grund, dass bei remove `updatedComment` steht und bei add `comment`?

Reply:
No it should be comment